Packs/IsItPhishing/Integrations/IsItPhishing/IsItPhishing.js

```diff
@@ -29,87 +29,92 @@ var sendRequest = function(method, api, body) {
 };
 
 var isPhishing = function(url, force, smart, area, timeout) {
-    var md;
-    var body = {
-        name: params.credentials ? params.credentials.identifier : params.name,
-        license: params.credentials  ? params.credentials.password : params.license,
-        version: '2',
-        force: force,
-        url: url,
-        area: area,
-        timeout: timeout
-    };
-    if (!area) {
-        delete body.area;
-    }
-    if (!timeout) {
-        delete body.timeout;
-    }
-    var res = sendRequest('POST', 'check', body);
-    var ec = {
-        IsItPhishing: {Url: url},
-        DBotScore: {
-            Indicator: url,
-            Score: 0,
-            Type: 'url',
-            Vendor: 'IsItPhishing',
-            Reliability: params.integrationReliability
+    var urls = url.split(',')
```

Comment:
we dont have any argtolist function? Is this how we're doing it in other js reputation commands?

Reply:
Yap